### PR TITLE
Post conflict resolution take two/zc

### PR DESCRIPTION
**Pull request by Yetidancer**
here goes nothin

### CODE REVIEW

File: lib/game_teams_collection.rb [hunk 1]
```diff
@@ -38,9 +38,7 @@ def game_teams_hash
   def home_games_only
     home_only = {}
     @game_teams_by_id.each do |team_id, games|
```
Comment (mronauli) [inline lib/game_teams_collection.rb:40]: i like it

File: lib/season_stats.rb [hunk 2]
```diff
@@ -5,9 +5,11 @@
 class SeasonStats
   include Createable
 
+  attr_reader :game_collection, :game_teams_collection, :season_game_teams_array, :season_game_hash
   def initialize(game_collection, game_teams_collection)
     @game_collection = game_collection
     @game_teams_collection = game_teams_collection
+    @season_game_hash = @game_collection.game_hash_from_array_by_attribute(@game_collection.games, :season)
```
Comment (mronauli) [inline lib/season_stats.rb:12]: efficiency!

File: test/season_stats_test.rb [hunk 3]
```diff
@@ -27,6 +27,69 @@ def test_it_can_report_head_to_head_results
     assert_equal expected, @season_stats.head_to_head_ids(16)
   end
 
+  def test_it_can_make_season_game_teams_array
+    assert_equal 12, @season_stats.make_season_game_array("20132014").length
+    # require "pry"; binding.pry
```
Comment (mronauli) [inline test/season_stats_test.rb:32]: stray pry

**Review by mronauli — COMMENTED** [review]
this is some serious work!!

File: lib/game_teams_collection.rb (hunk 1 above)
Comment (Yetidancer) [reply lib/game_teams_collection.rb:40]: gee thanks :)

File: test/season_stats_test.rb (hunk 3 above)
Comment (Yetidancer) [reply test/season_stats_test.rb:32]: yaaaaaa

**Pull request closed without merging**